src/options.js

```diff
@@ -0,0 +1,143 @@
+var BASE_URL = "https://api.susi.ai";
+//var messagesHistory=[];
+function showLoggedInBlock(show){
+	if(show){
+		noLoggedInBlock.style.display="none";
+		loggedInBlock.style.display="block";
+		document.getElementById("passwordchange").value="";
+		document.getElementById("passwordnewconfirm").value = "";
+		document.getElementById("passwordnew").value = "";
+	}
+	else{
+		noLoggedInBlock.style.display="block";
+		loggedInBlock.style.display="none";
+		document.getElementById("username").value = "";
+		document.getElementById("password").value = "";
+	}
+}
+
+themeSelect.addEventListener("change", function saveOptions(e) {
+	e.preventDefault();
+	var nameOfSettingsChanged = e.target.name;
+	if(nameOfSettingsChanged === "theme"){
+		// locally change the theme value
+		chrome.storage.sync.set({
+			theme: document.querySelector("#theme").value
+		});
+		
+		//check if user is logged in
+		if(accessToken!==""){
+
+			var selectedtheme = document.querySelector("#theme").value;
+
+			var url = BASE_URL+"/aaa/changeUserSettings.json?key1=theme&value1="+selectedtheme+"&accessTken="+accessToken+"&count=1";
+
+			// fire the api call to change settings value on server
+			$.ajax({
+
+				url: url,
+				dataType: "jsonp",
+				jsonpCallback: "q",
+				jsonp: "callback",
+				crossDomain: true,
+				success: function () {
+
+					alert("Theme updated successfuly.");
+
+				}
+
+			});
+
+		}
+		
+
+	}
+
+});
+loginForm.addEventListener("submit", function login(event){
+	event.preventDefault();
+	var email=document.getElementById("username").value;
+	var password=document.getElementById("password").value;
+	if(!email){
+		alert("Email can't be blank");
+		return;
+	}
+	else if(!password){
+		alert("Password can't be blank");
+		return;
+	}
+	loginButton.innerHTML="<i class='fa fa-spinner fa-spin fa-2x fa-fw'></i>";
+	var loginEndPoint = BASE_URL+"/aaa/login.json?type=access-token&login="+ encodeURIComponent(email)+ "&password="+ encodeURIComponent(password);
+	$.ajax({
+		url: loginEndPoint,
+		dataType: "jsonp",
+		jsonpCallback: "p",
+		jsonp: "callback",
+		crossDomain: true,
+		success: function (response) {
+			if(response.accepted){
+				accessToken = response.accessTken;
+				userEmail = email;
+				chrome.storage.sync.set({
+					loggedUser:{
+						email:email,
+						accessToken: accessToken
+					}
+				});
+
+
+				time = response.validSeconds;
+				loginButton.innerHTML="Login";
+				alert(response.message);
+				showLoggedInBlock(true);
+
+			}
+			else {
+				alert("Login Failed. Try Again");
+			}
+		},
+		error: function ( jqXHR) {
+			loginButton.innerHTML="Login";
+			var msg = "";
+			var jsonValue =  jqXHR.status;
+			if (jsonValue === 404) {
+				msg = "Login Failed. Try Again";
+			}
+			else {
+				msg = "Some error occurred. Try Again";
+			}
+			if (status === "timeout") {
+				msg = "Please check your internet connection";
+			}
+			alert(msg);
+		}
+	});
+
+});
+logoutButton.addEventListener("click", function logout(e){
```

Comment:
Logout button is not working! But I will fix all these issues in a PR.

Reply:
I have fixed it @ms10398